packages/block-editor/src/components/use-block-drop-zone/index.native.js

```diff
@@ -111,50 +116,82 @@ export default function useBlockDropZone( {
 	rootClientId: targetRootClientId = '',
 } = {} ) {
 	const targetBlockIndex = useSharedValue( null );
+	const dragPosition = {
+		x: useSharedValue( 0 ),
+		y: useSharedValue( 0 ),
+	};
+	const prevDragPosition = {
+		x: useSharedValue( 0 ),
+		y: useSharedValue( 0 ),
+	};
 
 	const { getBlockListSettings, getSettings } = useSelect( blockEditorStore );
 	const { blocksLayouts, getBlockLayoutsOrderedByYCoord } =
 		useBlockListContext();
 
 	const getSortedBlocksLayouts = useCallback( () => {
 		return getBlockLayoutsOrderedByYCoord( blocksLayouts.current );
+		// We use the value of `blocksLayouts` as the dependency.
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [ blocksLayouts.current ] );
```

Comment:
I perceive disabling the lint warning here is a stopgap to the more appropriate solution of updating the Hook dependencies to be `[ getBlockLayoutsOrderedByYCoord, blocksLayouts ]` long term. It makes sense to not make that long-term change alongside this bug fix. 👍🏻

Reply:
Yep, I agree. I merely added the comment to avoid the warning while applying changes to the file. I'd like to check further the implications of removing `blocksLayouts.current` dependency. I recall we wanted to memoize this call based on that, but I can't recall exactly why.